langchain4j-qianfan/src/main/java/dev/langchain4j/model/qianfan/QianfanChatModel.java

```diff
@@ -0,0 +1,137 @@
+package dev.langchain4j.model.qianfan;
+
+
+import dev.langchain4j.agent.tool.ToolSpecification;
+import dev.langchain4j.data.message.*;
+import dev.langchain4j.internal.Utils;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.qianfan.client.chat.ChatCompletionResponse;
+import lombok.Builder;
+import java.util.List;
+import java.util.Objects;
+import static dev.langchain4j.internal.RetryUtils.withRetry;
+import static dev.langchain4j.internal.Utils.getOrDefault;
+import static dev.langchain4j.model.qianfan.InternalQianfanHelper.*;
+import static java.util.Collections.singletonList;
+import dev.langchain4j.model.qianfan.client.chat.ChatCompletionRequest;
+
+/**
+ *
+ * see details here: https://cloud.baidu.com/doc/WENXINWORKSHOP/s/Nlks5zkzu
+ */
+
+public class QianfanChatModel implements ChatLanguageModel {
+
+
+
+    private final QianfanClient client;
+
+    private final String baseUrl;
+
+    private final Double temperature;
+    private final Float topP;
+    private final String modelName;
+
+    private final String endpoint;
+    private  final Float penaltyScore;
+    private final Integer maxRetries;
+
+    private final String responseFormat;
+
+
+    @Builder
+    public QianfanChatModel(String baseUrl,
+                            String apiKey,
+                            String secretKey,
+                            Double temperature,
+                            Integer maxRetries,
+                            Float topP,
+                            String modelName,
+                            String endpoint,
+                            String responseFormat,
+                            Float penaltyScore
+                             ) {
+        if (Utils.isNullOrBlank(apiKey)||Utils.isNullOrBlank(secretKey)) {
+            throw new IllegalArgumentException(" api key and secret key must be defined. It can be generated here: https://console.bce.baidu.com/qianfan/ais/console/applicationConsole/application");
+        }
+     
+        this.modelName=modelName;
+        this.endpoint=Utils.isNullOrBlank(endpoint)? QianfanModelEnum.getEndpoint(modelName):endpoint;
+
+        if (Utils.isNullOrBlank(this.endpoint) ) {
+            throw new IllegalArgumentException("Qianfan is no such model name. You can see model name here: https://cloud.baidu.com/doc/WENXINWORKSHOP/s/Nlks5zkzu");
+        }
+
+        this.baseUrl = getOrDefault(baseUrl,  "https://aip.baidubce.com");
+
+        this.client = QianfanClient.builder().baseUrl(this.baseUrl).apiKey(apiKey).secretKey(secretKey).logStreamingResponses(true).build();
```

Comment:
please add `logRequests` and `logResponses` properies to the builder so that user can configure it(instead of hardcoding it here), please check how it is done in OpenAI implementation